sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleCreateForm.java

```diff
@@ -101,8 +104,16 @@ protected void addFields() {
 		addField(SampleDto.SPECIMEN_CONDITION, ComboBox.class);
 		addField(SampleDto.NO_TEST_POSSIBLE_REASON, TextField.class);
 		addField(SampleDto.COMMENT, TextArea.class).setRows(2);
+		ComboBox labType = addField(SampleDto.LAB_TYPE, ComboBox.class);
 		CheckBox shipped = addField(SampleDto.SHIPPED, CheckBox.class);
 		CheckBox received = addField(SampleDto.RECEIVED, CheckBox.class);
+		
+		//hidding for labType
```

Comment:
Should the labtype column (which is now samplepurpose) have external as default value in the database?
because the field is required and the old records will be null if no default is set, which will cause unchecked errors

Reply:
Yes, all currently existing samples should be external since we didn't have any other sample purpose in the system until now.